components/locid/src/extensions/mod.rs

```diff
@@ -89,6 +90,7 @@ impl ExtensionType {
 /// A map of extensions associated with a given [`Locale`](crate::Locale).
 #[derive(Debug, Default, PartialEq, Eq, Clone, Hash, PartialOrd, Ord)]
 #[allow(missing_docs)] // TODO(#1028) - Add missing docs.
+#[allow(clippy::exhaustive_structs)] // this type is stable
```

Comment:
Nit: `non_exhaustive` as above. This is not a struct that people are expected to build themselves.